### PR TITLE
V2.16

### PR DESCRIPTION
**Pull request by jasonnovichRunAI**

### CODE REVIEW

**Comment by jasonnovichRunAI** [comment]
@lavianalon and @gal-revach Your requested changes are here.

File: docs/admin/runai-setup/cluster-setup/cluster-install.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ Using the cluster wizard:
 On the next page:
 
 * (SaaS and remote self-hosted cluster only) Install a trusted certificate to the domain entered above.
-*  Run the [Helm](https://helm.sh/docs/intro/install/) command provided in the wizard.
+* Run the [Helm](https://helm.sh/docs/intro/install/) command provided in the wizard.
 
 ## Cluster Table
```
Comment (gal-revach) [inline docs/admin/runai-setup/cluster-setup/cluster-install.md:29]: @jasonnovichRunAI I think the new sections should be after "Verify your installation"

File: docs/admin/runai-setup/cluster-setup/cluster-install.md [hunk 2]
```diff
@@ -38,7 +38,7 @@ The table provides the following columns:
 * **Status**&mdash;the status of the cluster. For more information see [Cluster status](#cluster-status). Hover over the information icon to see a short description and links to troubleshooting.
 * **Creation time**&mdash;the timestamp the cluster was created.
 * **URL**&mdash;the URL that was given to the cluster at the time of creation.
-* **Run:ai cluster version**&mdash;the version of the cluster.
+* **Run:ai cluster version**&mdash;the Run:ai version installed on the cluster.
 * **Run:ai cluster UUI**&mdash;the unique ID of the cluster.
 
 ### Cluster Status
```
Comment (gal-revach) [inline docs/admin/runai-setup/cluster-setup/cluster-install.md:44]: @jasonnovichRunAI I think the new sections should be after "Verify your installation"

File: docs/home/whats-new-2-16.md [hunk 3]
```diff
@@ -66,20 +48,22 @@ Fast, reliable, and easy-to-use unified interface.
 
 #### Authentication and Authorization
 
-* <!--  DONE RUN-13107/RUN-13108 - SSO users visibility-->Added a column to the *Users* table for the type of user that was created (Local or SSO). You can now also view the user's access role assignments directly from the *Users* table by pressing *VIEW*. In addition, SSO users who have logged into the system will now be visible in the *Users* table. For more information, see [Adding, Updating, and Deleting Users](../admin/admin-ui-setup/admin-ui-users.md#adding-updating-and-deleting-users).
+* <!--  DONE RUN-13107/RUN-13108 - SSO users visibility-->Added a column to the *Users* table for the type of user that was created (Local or SSO). You can now also view the user's access role assignments directly from the *Users* table by pressing *View*. In addition, SSO users who have logged into the system will now be visible in the *Users* table. For more information, see [Adding, Updating, and Deleting Users](../admin/admin-ui-setup/admin-ui-users.md#adding-updating-and-deleting-users).
```
Comment (gal-revach) [inline docs/home/whats-new-2-16.md:51]:
```suggestion
* <!--  DONE RUN-13107/RUN-13108 - SSO users visibility-->SSO users who have logged into the system will now be visible in the *Users* table. In addition, added a column to the *Users* table for the type of user that was created (Local or SSO). For more information, see [Adding, Updating, and Deleting Users](../admin/admin-ui-setup/admin-ui-users.md#adding-updating-and-deleting-users).
```

I removed the part about viewing the access rules of the user (it already existed before this version). Also changed the order from the major thing to the less important one..

**Review by gal-revach — COMMENTED** [review]
@jasonnovichRunAI please see my comments

**Pull request merged**